Blosc/config.py

```diff
@@ -0,0 +1,24 @@
+{
+
+	"downloads" : [
+
+		"https://github.com/Blosc/c-blosc/archive/v1.14.4.tar.gz"
+
+	],
+
+	"license" : "LICENSES/BLOSC.txt",
+
+	"commands" : [
+
+		"cmake -E make_directory {buildDir}/doc/licenses/blosc && cmake -E copy LICENSES/* {buildDir}/doc/licenses/blosc/",
+		"mkdir gafferBuild",
+		"cd gafferBuild &&"
+			" cmake"
+			" -G \"Unix Makefiles\""
+			" -D CMAKE_INSTALL_PREFIX={buildDir}"
+			" ..",
+		"cd gafferBuild && make install -j {jobs} VERBOSE=1"
```

Comment:
In the original build script there was this comment :

```
# Note : blosc does not declare its build dependencies
# correctly, so we cannot do a parallel build with `-j`.
```

Are you confident that this is no longer the case?

Reply:
Oh right I didn't see this one actually. I built it on a -j 40 machine and didn't see an issue appear. I did it one more time here and it worked.

Reply:
OK, let's assume it was fixed in a recent version then. If I hit problems doing the official builds then I'll fix it.